pkg/proxy/ipvs/proxier.go

```diff
@@ -629,9 +629,14 @@ func CanUseIPVSProxier() (bool, error) {
 	loadModules := sets.NewString()
 	wantModules.Insert(ipvsModules...)
 	loadModules.Insert(mods...)
-	modules := wantModules.Difference(loadModules).List()
-	if len(modules) != 0 {
-		return false, fmt.Errorf("Failed to load kernel modules: %v", modules)
+	modules := wantModules.Difference(loadModules)
+
+	// Now check if the missing kernel modules are there to be loaded
+	for kmod := range modules {
+		err := utilexec.New().Command("modprobe", "--dry-run", kmod).Run()
+		if err != nil {
+			return false, fmt.Errorf("Failed to load kernel module: %v", kmod)
```

Comment:
I think it should not return error directly here since it's a "best effort" way. Log warning instead?